### PR TITLE
Correct misspelling in `bench-scripts/unittests`

### PR DESCRIPTION
**Pull request by portante**

### CODE REVIEW

**Review by portante — COMMENTED** [review]
Ready!

**Review by webbnh — DISMISSED** [review]
🕺

File: README.md [hunk 1]
```diff
@@ -149,6 +149,15 @@ parameters to modify the test behavior:
   * `tox -- server legacy test-28 test-32`  -- run server legacy tests 28 & 32
   * `tox -- server python -v`               -- run server python tests verbosely
 
+For the `agent/bench-scripts` tests, one can run entire sub-sets of tests using
+and sub-directory name found in `agent/bench-scripts/tests`. For example:
```
Comment (npalaska) [inline README.md:153]: :s/using and/using a? :thinking:

**Review by npalaska — APPROVED** [review]

**Review by dbutenhof — APPROVED** [review]
That first one is quite a "misspelling"! 😆 

Seriously, that's a great discovery, although it's rather special-purpose since we don't have many (any?) tests outside bench-scripts that are functionally segregated like that.

**Review by npalaska — APPROVED** [review]

**Pull request merged**